.github/workflows/test.yml

```diff
@@ -3,13 +3,18 @@ on: push
 jobs:
   test:
     runs-on: ubuntu-latest
-    container: optechlab/indy-golang:1.16.0
     steps:
       - name: setup
         uses: actions/setup-go@v2
         with:
           go-version: 1.16.x
       - name: checkout
         uses: actions/checkout@v2
-      - name: test
-        run: make test
\ No newline at end of file
+      - name: install indy
+        run: ${GITHUB_WORKSPACE}/.github/workflows/install-indy.sh
+      - name: Run coverage
```

Comment:
Why not work thru `Makefile`? There was a similar rule if I recall correctly. However, makefile rules together with GitHub actions have helped my work earlier..
Actually, the same questions to other steps.